src/alloc_addresses/mod.rs

```diff
@@ -173,9 +132,9 @@ trait EvalContextExtPriv<'tcx>: crate::MiriInterpCxExt<'tcx> {
                     if memory_kind == MiriMemoryKind::Global.into() {
                         // For new global allocations, we always pre-allocate the memory to be able use the machine address directly.
                         let prepared_bytes = MiriAllocBytes::zeroed(info.size, info.align)
-                            .unwrap_or_else(|| {
-                                panic!("Miri ran out of memory: cannot create allocation of {size:?} bytes", size = info.size)
-                            });
+                        .unwrap_or_else(|| {
+                            panic!("Miri ran out of memory: cannot create allocation of {size:?} bytes", size = info.size)
+                        });
```

Comment:
Why did you un-indent this? Or did rustfmt do this automatically for some reason?

Reply:
Yes please undo the change. Good job tracking down the source of why rustfmt does not format this!